Luminary099/R63.agc

```diff
@@ -56,13 +56,13 @@
 # 6. GIMBAL ANGLES FROM VECPOINT TRANSFORMED INTO FDAI BALL ANGLES BY
 # BALLANGS.  FLASH DISPLAY V 06 N 18 AND AWAIT RESPONSE.
 #
-# 7 	RECYCLE -- RETURN TO STEP 4.
-#	TERMINATE -- EXIT R63
-#	PROCEED -- RESET 3AXISFLAG AND CALL R60LEM FOR ATTITUDE MANEUVER.
+# 7 	RECYCLE - RETURN TO STEP 4.
```

Comment:
Dot after 7 and alignment:

```
7. RECYCLE - RETURN TO STEP 4.
   TERMINATE - EXIT R63.
   PROCEED - RESET 3AXISFLG AND CALL R60LEM FOR ATTITUDE MANEUVER.
```

Reply:
Thanks for catching

Reply:
unclear regarding 'and alignment': Just add the dot or do you mean to use spaces instead of tabs, to match the scan's alignment relative to the "7."?

now:
```
7.<tab>RECYCLE ...
<tab>TERMINATE ...
```

changed:
```
7.<sp>RECYCLE ...
<sp><sp><sp>TERMINATE ... 
```